pkg/driver/provisioner_server_impl.go

```diff
@@ -125,126 +123,31 @@ func (s *ProvisionerServer) DriverCreateBucket(ctx context.Context,
 		return nil, status.Error(codes.InvalidArgument, "unsupported client type for bucket creation")
 	}
 
+	klog.V(3).InfoS("Creating bucket", "bucketName", bucketName)
```

Comment:
Can you use a constant or enum somewhere for log level so we know what 2, 3, 5 relates to

Reply:
Actually its a standard practice in kubernetes and klog.
[Kubernetes community guidelines](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#what-method-to-use)

Reply:
What do you think of 
```
const (
	LogLevelError = 0 // Error and critical logs
	LogLevelWarn  = 1 // Warnings
	LogLevelInfo  = 2 // General info messages
	LogLevelDebug = 3 // Debug-level messages
	LogLevelTrace = 5 // Detailed trace logs
)
```

Reply:
Added in the latest review commit: https://github.com/scality/cosi-driver/pull/67/commits/b2dfa498fffeb21d18c2af76e6696ccc9e6e810d